infra/scripts/seed-roles.sh

```
@@ -0,0 +1,92 @@
+DB_PASSWORD=${DB_PASSWORD:-hotakupassword}
+DB_NAME=${DB_NAME:-hotaku_db}
+
+# Colors for output
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Function to print colored output
+print_status() {
+    if [ $1 -eq 0 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)